ci/run.sh

```diff
@@ -21,6 +21,3 @@ make tests
 make install
 cd bin
 ./unit_tests
-
-cd ../../tests/
-python3 runner.py
```

Comment:
Is this removing the execution of tests from travis?

Reply:
No, they are run by `make tests`. @clanmills added them to the test makefile as `newtests`